src/core/cpu_patches.cpp

```diff
@@ -339,9 +356,26 @@ static void GenerateBLSR(const ZydisDecodedOperand* operands, Xbyak::CodeGenerat
 
     SaveRegisters(c, {scratch});
 
+    Xbyak::Label set_carry, clear_carry, end;
```

Comment:
With the other suggestions, the `set_carry` variable could be removed from here.